package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-multipart",
-  "version": "2.0.0",
```

Comment:
Please restore the previous value.

Our release tool will update this field automatically (and it will break if we do it manually 😅 )

Reply:
Line reverted.